.github/workflows/image_rs_build.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       fail-fast: false
       matrix:
         rust:
-          - 1.72.0
+          - 1.76.0
```

Comment:
IIRC we agreed (https://github.com/confidential-containers/confidential-containers/issues/193) to use 'latest stable' with the explicit version pinned. Trustee uses 1.76.0 which was the 'latest stable' back when this topic was discussed but is also outdated.

Reply:
> I guess you would lobby for us to bump image-rs to 1.78, trustee to 1.76 and unpinned for the other guest components?

Given Kata uses a single version and all of these are consumed by Kata (or its CI), perhaps it's good to have a unified version specified. Initially my thinking was that following the latest stable is enough but that may not be a good choice after all. Thinking about more, would it make sense to have a clear 'MSRV policy' defined for CoCo which set guidelines for ourselves and downstream users?

Reply:
Sorry to mislead - Kata containers has individual versions for trustee, guest-components (AA, CDH, ASR) and the kata-agent (image-rs & ocicrypt-rs) as we had to split them out to get things to build, however divergent versions does lead to some confusion and complications with CVEs. We'll see what the discussion today brings and I'll report back here with a summary.

An alternative approach is that we could consider publishing binaries (via oras packages) for the guest-components and trustee in confidential-containers and then just consume them directly in kata-containers rather than building them there, which might help reduce some of the dependencies, but will probably introduce other issues!

Reply:
Yes - I also was referring to the rust version too, so my clarification didn't help either! 😿

Reply:
So we didn't get a great response from the Kata AC - Zvonko needs to bump the version to at least 1.74, but there is some resistance from RH folks to stay latest stable at the moment as they have an old version of rust in their base image. We've asked them to go and investigate and see what version they can bump to and then get back to us.